widgetssdk/src/main/res/layout/single_choice_card_view.xml

```diff
@@ -17,35 +17,35 @@
             android:id="@+id/layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginHorizontal="@dimen/glia_large"
-            android:layout_marginTop="@dimen/glia_medium"
-            android:layout_marginBottom="@dimen/glia_large">
+            android:layout_marginTop="@dimen/glia_pre_large"
+            android:layout_marginBottom="@dimen/glia_pre_large">
 
-            <ImageView
+            <com.google.android.material.imageview.ShapeableImageView
                 android:id="@+id/image"
                 android:layout_width="0dp"
-                android:layout_height="wrap_content"
+                android:layout_height="@dimen/glia_single_choice_card_image_height"
+                android:layout_marginHorizontal="@dimen/glia_large"
                 android:adjustViewBounds="true"
                 android:scaleType="centerCrop"
-                android:visibility="gone"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
+                app:layout_constraintTop_toTopOf="parent"
+                app:shapeAppearanceOverlay="@style/Application.Glia.Chat.ShapeAppearance.SmallComponent"
+                tools:ignore="ContentDescription"
+                tools:src="@tools:sample/backgrounds/scenic" />
 
             <com.glia.widgets.view.textview.ChoiceCardContentTextView
                 android:id="@+id/content_view"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/glia_large"
                 android:layout_marginTop="@dimen/glia_medium"
-                android:gravity="center"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@id/image"
                 app:layout_constraintWidth_default="spread"
-                app:layout_goneMarginTop="0dp" />
-
+                app:layout_goneMarginTop="0dp"
+                tools:text="What type of insurance are you interested in?" />
```

Comment:
just wanted to clarify, where this text will be visible?

Reply:
It's visible only in android studio design tool